tests/test_georaster.py

```diff
@@ -72,12 +72,24 @@ def test_info(self):
             assert r.__getattribute__(attr) == r.ds.__getattribute__(attr)
 
         # Check summary matches that of RIO
-        assert print(r) == print(r.info())
+        assert str(r) == r.info()
 
         # Check that the stats=True flag doesn't trigger a warning
         with warnings.catch_warnings():
             warnings.simplefilter("error")
-            r.info(stats=True)
+            stats = r.info(stats=True)
+
+        r.data.ravel()[:1000] = 0
```

Comment:
Thanks for adding it! I would just add a quick comment here to explain what you do here.
Also, I believe the minimum should be checked, rather than maximum (the latter will not change if you set values to 0...).

Reply:
Yes, my bad! Fixed in 10146b2